app/controllers/pages_controller.rb

```diff
@@ -63,12 +63,9 @@ def latest_published_welcome_thread
 
   def members_for_display
     Rails.cache.fetch("members-for-display-on-membership-page", expires_in: 6.hours) do
-      members = User.with_any_role(:level_1_member,
-                                  :level_2_member,
-                                  :level_3_member,
-                                  :level_4_member,
-                                  :triple_unicorn_member,
-                                  :workshop_pass)
+      roles = %i[level_1_member level_2_member level_3_member level_4_member triple_unicorn_member
+                 workshop_pass]
+      members = User.select(:id, :username, :profile_image).with_any_role(*roles)
```

Comment:
Only need `id`, `username`, and `profile_image`. Hopefully profile_image can be cached without issue.